src/id_token.erl

```diff
@@ -31,7 +31,8 @@ sign(Alg, Claims) ->
   end.
 
 refresh_and_validate(Provider, IdToken) ->
-  #{keys := FreshKeys} = id_token_provider:refresh_keys(Provider),
+  #{keys := FreshKeys} =
+    id_token_provider:refresh_keys(Provider, #{force_refresh => true}),
```

Comment:
not sure I understand this correctly, but isn't this the only place where `id_token_provider:refresh_keys` is called? why do you need the `maybe_refresh` function at all then?